mofa/model.py

```diff
@@ -148,6 +148,17 @@ class LigandDescription:
     def atoms(self):
         return read_from_string(self.xyz, "xyz")
 
+    def find_anchor_C_idx(self) -> list:
```

Comment:
Does @hyunp2 need the carbons before or after we've run the generator? This part is useful for after